block-producer/src/state_view/mod.rs

```diff
@@ -162,6 +174,12 @@ fn ensure_in_flight_constraints(
     Ok(())
 }
 
+#[instrument(
+    target = "miden-block-producer",
+    skip(candidate_tx, tx_inputs),
+    err,
+    fields(tx_inputs.account_hash = %format_opt(tx_inputs.account_hash.as_ref()), nullifiers = ?tx_inputs.nullifiers))
```

Comment:
I'm not sure there is much value in adding `account_hash` and `nullifiers` as fields here. If we want to log them I'd do that as a message inside the function at `DEBUG` level.